src/config.go

```diff
@@ -60,3 +67,69 @@ func LoadConfig() error {
 
 	return nil
 }
+
+func ComposeUrl() string {
+
+	protocol := "http"
+	if cfg.TLS {
+		protocol = "https"
+	}
+
+	return fmt.Sprintf("%v://%v:%v", protocol, cfg.Host, cfg.Port)
+
+}
+
+// Update the token in the configuration file
+func UpdateConfigToken(token string) error {
+
+	path, err := GetConfigPath()
+	if err != nil {
+		return err
+	}
+
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return err
+	}
+
+	token_found := false
+
+	lines := strings.Split(string(content), "\n")
+
+	for i, line := range lines {
+		if strings.HasPrefix(line, "token:") {
+			lines[i] = "token: " + token
+			token_found = true
+		}
+	}
```

Comment:
I agree, it would be be better. However the Go yaml library does not support the preservation of comments.

https://github.com/go-yaml/yaml/issues/132